src/test/scala-2.12/org/apache/james/gatling/simulation/Configuration.scala

```diff
@@ -20,8 +20,13 @@ object Configuration {
   val WEBADMIN_PROTOCOL = Properties.envOrElse("WEBADMIN_PROTOCOL", "http")
   val BaseJamesWebAdministrationUrl = new URL(s"$WEBADMIN_PROTOCOL://$WebadminServerHostName:$WEBADMIN_PORT")
 
-  val ScenarioDuration = 1 hour
-  val InjectionDuration = 1 hour
+  val DURATION_PROPERTY = Properties.envOrNone("DURATION") match {
+    case Some(duration) => Some(duration.toInt minutes)
+    case _ => None
+  }
+
+  val ScenarioDuration = DURATION_PROPERTY.getOrElse(1 hour)
+  val InjectionDuration = DURATION_PROPERTY.getOrElse(1 hour)
```

Comment:
Keeping both for now until I get this to work but wondering if should keep both... There is few differences in the code obviously but I'm still not sure exactly what was initially intended here.

Reply:
Inject say 1.000 users over 1 minutes then play a simulation with all of them (loop) for 1 hour. 

Let's keep this.